src/Arcus.Testing.Tests.Unit/Assert_/AssertCsvTests.cs

```diff
@@ -240,6 +240,72 @@ public void CompareWithIgnoredRowAndColumnOrder_WithShuffledCsv_Succeeds()
             });
         }
 
+        [Property]
+        public void CompareWithIgnoredColumnOrderAndColumnIndexes_WithShuffledCsv_FailsWithDescription()
+        {
+            // Arrange
+            TestCsv expected = TestCsv.Generate();
+            TestCsv actual = expected.Copy();
+
+            actual.ShuffleColumns();
+
+            CompareShouldFailWithDescription(
+                expected,
+                actual,
+                options =>
+                {
+                    options.ColumnOrder = AssertCsvOrder.Ignore;
+                    options.IgnoreColumn(expected.IgnoredIndex);
+                },
+                "cannot compare", "column indexes", "column order", "included",
```

Comment:
```suggestion
                "cannot compare", "indexes", "column order", "included",
```
The tests fail bc my suggestion included 'columns' before indexes. This will fix it.